### PR TITLE
Coding style part1

### PR DESCRIPTION
**Pull request by kintel**
This is as small of a patch I can meaningfully produce.
It still does some aggressive indentation some places, but I don't know if that can be dealt with; at least all indenting should be auto-generate now.

Generated by `./scripts/beautify.sh --all --uncrustify`

### CODE REVIEW

**Comment by kintel** [comment]
@justbuchanan

File: src/parsersettings.h [hunk 1]
```diff
@@ -13,6 +13,6 @@ extern int parser_error_pos;
 void parser_init();
 
 fs::path search_libs(const fs::path &localpath);
-fs::path find_valid_path(const fs::path &sourcepath, 
-                         const fs::path &localpath, 
-                         const std::vector<std::string> *openfilenames = nullptr);
+fs::path find_valid_path(const fs::path &sourcepath,
+												 const fs::path &localpath,
+												 const std::vector<std::string> *openfilenames = nullptr);
```
Comment (justbuchanan) [inline src/parsersettings.h]: This indent seems a little large.

**Comment by justbuchanan** [comment]
Looks good! I like that uncrustify has an "ignore" option so you can direct it to only fix certain types of style mistakes.

I scanned through the diff and overall it looks good. I noticed one minor indenting issue in src/parsersettings.h (and possibly other places) where it added a large indent before function parameter declarations. Not a big deal, but maybe there's a related option to adjust?

**Comment by thehans** [comment]
I don't have anything to add about specific settings, but I saw this and thought it might be pretty handy if you are trying to tweak very specific indenting options: http://universalindent.sourceforge.net/features.php

File: src/parsersettings.h (hunk 1 above)
Comment (kintel) [reply src/parsersettings.h]: It's because the github web viewer default to a tab size of 8, while i use 2.

Comment (kintel) [reply src/parsersettings.h]: ..which is one of the reasons why we shouldn't have used tab for indenting in the first place : /

Comment (kintel) [reply src/parsersettings.h]: https://github.com/openscad/openscad/blob/e1c1ba37fa717f882cc44a7eac70be9a83a1dd7e/src/parsersettings.h?ts=2

**Comment by kintel** [comment]
@justbuchanan Unfortunately, the ignore option doesn't work very well, I tried to ignore what I could to keep the patch small. If you see anything I missed I'd be interested.

**Comment by kintel** [comment]
@thehans Universalindent looks slightly out of maintenance. Both uncrustify and clang-format are actively developed tools; we just need to establish a good config.

**Pull request merged**